config/experiment.yaml

```diff
@@ -0,0 +1,49 @@
+---
+experiment:
+  pipeline: Pooled Cell Painting
+  project_tag: Periscope
+  batch: BATCH_ID
+  plates:
+    - PLATE01
+    - PLATE02
+  compartments:
+    - Cells
+    - Nuclei
+    - Cytoplasm
+  control_barcode_ids:
+    - NT
+  cell_id_cols:
+    - ImageNumber
+    - ObjectNumber
+  cell_match_cols:
+    cells:
+      - Parent_Nuclei
+    cytoplasm:
+      - Parent_Nuclei
+      - Parent_Cells
+    spots:
+      - Parent_Cells
+  categorize_cell_quality: simple
+  cell_filter:
+    - Perfect
+    - Great
+  ignore_files:
+    - .DS_Store
+---
+directory_structure:
+  input_data_dir: /Users/gway/work/projects/2018_11_20_Periscope_Calico/workspace/analysis
+  plate_id_folder: true
+  output_data_dir: data/
+  preprocessing:
+    base: 0.site-qc/
+    plate_id_folder: true
+    spots: spots/
+    paint: paint/
+    data: data/
+    results: results/
+    figures: figures/
+  profiling:
+    base: 1.profiles/
+    plate_id_folder: true
+    single_cell: single_cell/
+    profiles: profiles/
```

Comment:
> If we're conceptualizing experiment.yaml as the file that users will be continually interacting with and options.yaml as the "advanced case" options that will be infrequently (if ever) adjusted, I have some questions/suggestions/thoughts:

Ah, good point that I should clarify. I actually did not conceptualize these two files in this way. I was thinking:

| File | Purpose |
| :--- | :-------- |
| `experiment.yaml` | Define general experimental parameters that are not specific to any recipe step and setup directory/file structures |
| `options.yaml` | Provide options for each step of the two recipe modules in one file.

I envision that the `advanced_options.yaml` will be entirely broken off from `options.yaml` at a later date.

Reply:
> Currently in experiment.yaml, experiment:, compartments: will almost never change. Even if an experiment were to label more and more compartments, the group has historically found that object identification of further compartments is too variable so this can move to options.yaml
Likewise, unless someone were to do some pretty funky manipulation of the CellProfiler pipelines/workflow, cell_id_columns and cell_match_columns won't change so they should move to options.yaml as well.

Cool. I'll move these things to a "core" options page in the `options.yaml`. 

> Conversely, in options.yaml, preprocess:, summarize-plate:, barcoding_cycles definitely moves around between experiments and should be in experiment.yaml.

Love it!! Yes

> How does the cell_filter set in experiment: relate to filter_cell_quality set in profile:? Will it be a problem if there is a mismatch between them? If so, they should both be in experiment.yaml.

Nice catch. They're redundant. I will remove in `options.yaml`